docs/source/commandline/commandline-all.rst

```diff
@@ -1203,18 +2044,33 @@ Core Video Options
       |	The default on all other systems is ``soft``.
       |
 
+    Example:
+        .. code-block:: bash
+
+            mame64 gradius3 -video bgfx
+
 .. _mame-commandline-numscreens:
 
 **-numscreens** *<count>*
 
     Tells MAME how many output windows to create.  For most systems, a single
-    output window is all you need, but some systems originally used multipl
+    output window is all you need, but some systems originally used multiple
     screens (*e.g. Darius and PlayChoice-10 arcade machines*).  Each screen
     (up to 4) has its own independent settings for physical monitor, aspect
     ratio, resolution, and view, which can be set using the options below.
```

Comment:
Maybe add a note that systems with front panel controls and status lights may let you put different parts of the system in different windows, etc.

Reply:
It's in there.